docs/src/extend/custom-processors.md

```diff
@@ -140,7 +140,56 @@ By default, ESLint does not perform autofixes when a custom processor is used, e
 
 You can have both rules and custom processors in a single plugin. You can also have multiple processors in one plugin. To support multiple extensions, add each one to the `processors` element and point them to the same object.
 
-**The `meta` object** helps ESLint cache the processor and provide more friendly debug message. The `meta.name` property should match the processor name and the `meta.version` property should match the npm package version for your processors. The easiest way to accomplish this is by reading this information from your `package.json`.
+### How `meta` Objects are Used
+
+The `meta` object helps ESLint cache configurations that use a processor and to provide more friendly debug messages. The `meta.name` property should match the processor name and the `meta.version` property should match the npm package version for your processors. The easiest way to accomplish this is by reading this information from your `package.json`.
```

Comment:
"The meta.name property should match the processor name" - this applies only to processor's own meta, so perhaps it should be in that section.

Reply:
updated